fleet_vehicle_service_activity/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Create an activity to vehicle fleet manager days before service date",
+    "summary": """ Set days before service date on fleet settings and
+                   create an activity to vehicle fleet manager""",
```

Comment:
Don't add line breaks and spaces on the summary, and it has to be shorter:

```suggestion
    "summary": "Activity alerts for fleet services",
```